mailpoet/assets/css/src/components-automation-integrations/woocommerce.scss

```diff
@@ -19,3 +19,10 @@
     background: transparent;
   }
 }
+
+.mailpoet-product-rating-range {
+  input {
+    margin: 0 5px;
+    max-width: 50px;
+  }
+}
```

Comment:
What are these styles for? If it's for something in premium, why not put them there?

Reply:
Moved into Premium